.github/workflows/tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.x
+          python-version: "3.9 - 3.10"
```

Comment:
Why not just 3.10?

Reply:
I thought it might make it faster if we specified "use Python 3.9 or 3.10, I don't care". But actually, both should be in the runner's tool cache anyway. So it shouldn't matter. I'll change it so it's just 3.10 :)